### ISSUE TITLE
videomae support frame size of 8

### ISSUE THREAD

**Issue by zhudongwork** [MCG-NJU/VideoMAE#58]
The frame size seen from the experiment of the paper is 16 or 32, does videomae support frame size of 8?

**Comment by TitaniumOne** [comment]
> The frame size seen from the experiment of the paper is 16 or 32, does videomae support frame size of 8?

The same problem for me, do you have any ideas?

My GPUs are limited，I need less frames(=8) and bigger batch size( > 16) as input.
If input 8 frames, then the error comes like `The size of tensor a (x) must match the size of tensor b (y) at non-singleton dimension` at ```
x = self.patch_embed(x)
x = x + self.pos_embed.type_as(x).to(x.device).clone().detach()
```

**Comment by JinChow** [comment]
@TitaniumOne Yeah, I have the same problem, the error comes like The size of tensor a (x) must match the size of tensor b (y) at non-singleton dimension at 
x = self.patch_embed(x)
x = x + self.pos_embed.type_as(x).to(x.device).clone().detach()
Have you solve the problem? I will appreciate it if you can help me ! Thank you!
